### PR TITLE
updated setup.py with missing segmentation module

### PR DESCRIPTION
**Pull request by sb43**
Added additional output files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,25 +48,31 @@ Various exceptions can occur for malformed input files.
 
   following tab separated output files were produced
 
- 1. crispr_cleanr_normalised_counts.tsv
+ 1. normalised_counts.tsv
```
Comment (ghost) [inline README.md:53]: Lines 14-15 links are broken

File: pyCRISPRcleanR/crisprCleanR_command.py [hunk 2]
```diff
@@ -52,8 +52,8 @@ def main():  # pragma: no cover
     optional.add_argument("-np", "--num_processors", type=int, dest="num_processors", required=False,
                           default=1, help="Number of processors to use for parallel jobs")
 
-    optional.add_argument("-cc", "--correct_counts", type=int, dest="correct_counts", required=False,
-                          default=None, help="Correct counts using CBS [Y/y]")
+    optional.add_argument("-cc", "--crispr_cleanr", type=str, dest="crispr_cleanr", required=False,
```
Comment (ghost) [inline pyCRISPRcleanR/crisprCleanR_command.py]: Could this have been a flag rather than having a value? `action='store_true'`. Non essential, just a query

**Review by ghost — CHANGES_REQUESTED** [review]
Have you considered a requirements.txt file for building the virtual env and describing requirements? 
Using 
`pip freeze | grep -v 'echo ${PWD##*/}' > requirements.txt`

```
hash virtualenv || pip3 install virtualenv
virtualenv -p python3 env
source env/bin/activate
pip install -r requirements.txt
```

File: pyCRISPRcleanR/crisprCleanR_command.py (hunk 2 above)
Comment (sb43) [reply pyCRISPRcleanR/crisprCleanR_command.py]: made this change

File: README.md (hunk 1 above)
Comment (sb43) [reply README.md:53]: links now fixed

**Pull request merged**